src/core/CMakeLists.txt

```diff
@@ -49,18 +46,16 @@ endif()
 # Allow CLOG to preprocess all the source files.
 CLOG_GENERATE_TARGET(CORE_CLOG_LIB ${SOURCES})
 
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${QUIC_C_FLAGS}")
+target_link_libraries(CORE_CLOG_LIB PRIVATE inc)
```

Comment:
What do you think about renaming all these *_CLOG_LIB libraries to *.clog instead?

Reply:
Easy enough. I can do that.